packages/libcap.rb

```diff
@@ -2,20 +2,12 @@
 
 class Libcap < Package
   description 'Libcap implements the user-space interfaces to the POSIX 1003.1e capabilities available in Linux kernels.'
-  homepage 'https://directory.fsf.org/wiki/Libcap'
-  version '2.66'
+  homepage 'https://directory.fsf.org/wiki/Libcap/'
+  version '2.70'
   license 'GPL-2 or BSD'
   compatibility 'all'
-  source_url 'https://git.kernel.org/pub/scm/libs/libcap/libcap.git'
+  source_url 'https://kernel.googlesource.com/pub/scm/libs/libcap/libcap.git'
   git_hashtag "libcap-#{version}"
```

Comment:
It looks like the `libcap-2.70` and the `v1.2.70` tags are identical: 

- https://git.kernel.org/pub/scm/libs/libcap/libcap.git/tag/?h=v1.2.70
- https://git.kernel.org/pub/scm/libs/libcap/libcap.git/tag/?h=libcap-2.70

Although, not sure if `v1.#{version}` is more readable or makes more sense than `libcap-#{version}`